### PR TITLE
ZH-trans: add aws.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/424) has no assignee. will not be reported to the task issues

File: content/zh/docs/setup/turnkey/aws.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+<!--
+For more complete applications, please look in the [examples directory](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/)
+-->
+有关更完整的应用程序，请查看[示例目录](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/)
+
+<!--
+## Scaling the cluster
+-->
+
+## 扩展的集群
```
Comment (tengqm) [inline content/zh/docs/setup/turnkey/aws.md]:
```suggestion
## 扩展集群
```

File: content/zh/docs/setup/turnkey/aws.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+## Tearing down the cluster
+-->
+
+## 拆除集群
+
+<!--
+Make sure the environment variables you used to provision your cluster are still exported, then call the following script inside the
+`kubernetes` directory:
+-->
+确保用于提供集群的环境变量在导出，然后在 `kubernetes` 目录中调用以下脚本：
```
Comment (tengqm) [inline content/zh/docs/setup/turnkey/aws.md]:
```suggestion
确保用于提供集群的环境变量已经导出，然后在 `kubernetes` 目录中调用以下脚本：
```

Comment (xichengliudui) [reply content/zh/docs/setup/turnkey/aws.md]: 已修改。

**Pull request merged**